File: 251025/남녀노소 구분짓기/sex-and-age.js
const fs = require('fs');

let input = fs.readFileSync(0).toString().trim().split('\n');
let result = null;

let human = {
    gender : Number(input[0]),
    age : Number(input[1]),
};

if (human.gender === 0) {
    if (human.age >= 19) {
        result = 'MAN';
    } else {
        result = 'BOY';
    }
} else {
    if (human.age >= 19) {
        result = 'WOMAN';
    } else {
        result = 'GIRL';
    }
}

console.log(result);

File: trail0/남녀노소 구분짓기/sex-and-age.cpp
#include <iostream>
using namespace std;

int main() {
    int gender, age;
    string result;
    cin >> gender >> age;
    if (gender == 0) {
        result = age >= 19 ? "MAN" : "BOY";
    } else {
        result = age >= 19 ? "WOMAN" : "GIRL";
    }
    cout << result;
    return 0;
}